maven-plugin-plugin/src/it/settings.xml

```diff
@@ -39,9 +39,11 @@ under the License.
           <url>@localRepositoryUrl@</url>
           <releases>
             <enabled>true</enabled>
+            <checksumPolicy>ignore</checksumPolicy>
           </releases>
           <snapshots>
             <enabled>true</enabled>
+            <checksumPolicy>ignore</checksumPolicy>

```

Comment:
why are we ignoring checksums?

Reply:
Some tests fail with the following exception:
```
Caused by: org.eclipse.aether.resolution.ArtifactResolutionException: Could not transfer artifact org.apache.maven.plugin-tools:maven-script-ant:pom:3.7.1-SNAPSHOT from/to local.central (file:///Users/gnodet/.m2/repository): Checksum validation failed, no checksums available
```

Reply:
Ah, well, local repository should not be ..... :smile: